util/src/main/java/tc/oc/pgm/util/tablist/TabManager.java

```diff
@@ -141,11 +140,12 @@ public void onWorldChange(PlayerChangedWorldEvent event) {
     if (view != null) view.onWorldChange(event);
   }
 
-  @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
-  public void onSkinPartsChange(PlayerSkinPartsChangeEvent event) {
-    TabEntry entry = this.getPlayerEntryOrNull(event.getPlayer());
-    if (entry instanceof PlayerTabEntry) {
-      ((PlayerTabEntry) entry).onSkinPartsChange(event);
-    }
-  }
+  // TODO: This is not in spigot
+  //  @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
+  //  public void onSkinPartsChange(PlayerSkinPartsChangeEvent event) {
+  //    TabEntry entry = this.getPlayerEntryOrNull(event.getPlayer());
+  //    if (entry instanceof PlayerTabEntry) {
+  //      ((PlayerTabEntry) entry).onSkinPartsChange(event);
+  //    }
+  //  }
```

Comment:
This does not seem to be necessary to support, logging out/in is reasonable to expect for a refresh

Reply:
A re-login will not necessarily cause an update. Skin flags may be sent after login with a settings packet, that would likely be ignored. The result from not having this is no overlay skin layer ever showing in tab